paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/AuBecsDebitDefinition.kt

```diff
@@ -12,7 +12,7 @@ import com.stripe.android.ui.core.elements.SharedDataSpec
 internal object AuBecsDebitDefinition : PaymentMethodDefinition {
     override val type: PaymentMethod.Type = PaymentMethod.Type.AuBecsDebit
 
-    override val supportedAsSavedPaymentMethod: Boolean = true
+    override val supportedAsSavedPaymentMethod: Boolean = false
```

Comment:
I mapped what we were doing before to this (from PaymentMethodRequirements). But when I started using it, I intended for it to mean something different.

So now all the values reflect what they should be, rather than how they were before.